README.md

```diff
@@ -36,6 +36,20 @@ This chart creating project was built using:
   `REACT_APP_OPENWEATHER_KEY='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'`  
   You should take the KEY from the developer!
 
+* Project deployment ([firebase.google.com](https://firebase.google.com)) :
+  - To host your site with Firebase Hosting, you need the Firebase CLI (a command line tool).  
+    Run the following npm command to install the CLI or update to the latest CLI version.  
+    `npm install --global firebase-tools`;
+  - Sign in to Google.  
+    For deploy new project you should create new firebase project.  
+    For redeploy current project you must be added to members of current firebase project.  
+    `firebase login`;
+  - Build your web app.  
+    `npm run build`;
+  - Deploy or redeploy your web app.  
+    Put your static files (e.g., HTML, CSS, JS) in your app's deploy directory (the default is "public").  
+    `firebase deploy`.
+
```

Comment:
As a next step I could propose to automate the process using GitHub actions. so you could deploy your app automatically after a merge to main branch
Here is a tutorial: https://www.youtube.com/watch?v=QMATr0vCsYg

Reply:
Done!